data/widgets/tooltips.ui

```diff
@@ -47,7 +47,7 @@
       <object class="GtkLabel" id="license-label">
         <property name="visible">True</property>
         <property name="can_focus">False</property>
-        <property name="label" translatable="yes">View the license in your browser</property>
+        <property name="label" translatable="yes">View the license in a new window</property>
```

Comment:
I'd say simply "View the license"

In any case, this one I think we'll have to skip cherry picking to eos3.0 unless we order another round of translations for 3.0.5.